modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/metric/SqlViewExporterSpiTest.java

```diff
@@ -560,7 +560,8 @@ public void testContinuousQuery() throws Exception {
         }
 
         assertTrue(execute(ignite0, "SELECT * FROM SYS.CONTINUOUS_QUERIES").isEmpty());
```

Comment:
Should we use the same `waitForCondition` as well?

Reply:
For local node QueryCursor#close gives guarantees that CQ information will be removed before method returns. See GridContinuousProcessor#stopRoutine::1094